config/environments/development.rb

```diff
@@ -57,7 +57,7 @@
   # Debug mode disables concatenation and preprocessing of assets.
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
-  config.assets.debug = true
+  config.assets.debug = false
```

Comment:
I think this should still default to true? https://github.com/rails/rails/blob/3446efc8bca2cb310aabc4b2e801598252344e0e/guides/source/configuring.md#configuring-assets